.travis.yml

```diff
@@ -1,32 +1,24 @@
-matrix:
-  include:
-    - os: linux
-      env: BUILD_TYPE=Debug   STD=c++0x
-    - os: linux
-      env: BUILD_TYPE=Debug   STD=c++11
-    - os: linux
-      env: BUILD_TYPE=Release STD=c++0x
-    - os: linux
-      env: BUILD_TYPE=Release STD=c++11
-    - os: osx
-      env: BUILD_TYPE=Debug   STD=c++11
-    - os: osx
-      env: BUILD_TYPE=Release STD=c++11
-
-language:
-    - cpp
+language: cpp
```

Comment:
It didn't show up before. Here is the last build of master (with the old config) https://travis-ci.org/google/benchmark.

I was hoping a different syntax would make it show up, but it doesn't.